app/src/main/java/com/example/returnpals/composetools/PricingUI.kt

```diff
@@ -50,17 +50,18 @@ import kotlinx.coroutines.flow.update
  * Draws the entire screen of the step "Choose PricingPlan" in the "Schedule a Return" process
  */
 @Composable
-fun ScheduleReturn.PricingUI(
+fun PricingUI(
     modifier: Modifier = Modifier,
-    navController: NavController? = null,
-    onChangePlan: (PricingPlan) -> Unit = {},
-    plan: PricingPlan? = null,
+    onChangePlan: (PricingPlan) -> Unit,
+    onClickNext: () -> Unit,
+    onClickBack: () -> Unit,
+    plan: PricingPlan?,
     guest: Boolean = false,
 ) {
     ScheduleReturnScaffold(
         step = 3,
-        onClickNext = { /*TODO: navigate to package details */ },
-        onClickBack = { /*TODO: navigate to pickup method */ },
+        onClickNext = onClickNext,
```

Comment:
The TODO you made "navigate to package details" was also very helpful.  We can preserve that information by changing the parameter name. Next time, try changing onClickNext to something like "navToPackageDetails".

Reply:
I like it's current name as it describes the event that the lambda will be called on. The behavior of that button is outsourced for another function to decide, which doesn't necessarily have to be navigate to package details and we can't enforce that specific behavior too so I like the name to describe the event not the expected behavior.